### ISSUE TITLE
Error saving .rq file

### ISSUE THREAD

**Issue by pwolfrum** [zazuko/vscode-sparql-notebook#69]
I loaded a sparql query from xyz.rq to a notebook cell. Executing the cell and displaying the retrieved content works fine.
When I save the notebook, it should automatically save all connected .rq files. 

It tries to do this and replies with
"Error writing file: xyz.rq: NoPermissions (FileSystemError): Error: EPERM: operation not permitted, open 'C:\xyz.rq'"

Very strange it is trying to save to C:\, althoug both the notebook and xyz.rq reside in differnt directories (to which VSCode can write)

**Comment by BenjaminHofstetter** [comment]
@pwolfrum yes i would like to help you. I because i don't have access to a windows system i did never try it there. Can you give me a sample file ? I would like to check the saved paths.

**Comment by pwolfrum** [comment]
Thanks a lot for fixing this! And sorry I overlooked your question in
December!

Regards,
Philipp


Am Mi., 10. Apr. 2024 um 12:11 Uhr schrieb Benjamin Hofstetter <
***@***.***>:
